examples/using-wordpress/src/layouts/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class DefaultLayout extends React.Component {
```

Comment:
SFC?
```js
const DefaultLayout = ({ children }) => (
  <div>{children}</div>
);
```

Also can you get away without the `div` wrapper by returning `React.Children.only(children)`?